### ISSUE TITLE
Jaws does not read text brought dynamically to the page

### ISSUE THREAD

**Issue by amad4biz** [FreedomScientific/VFO-standards-support#68]
### Summary

> Brief description of the issue. Please list any specific steps.
Jaws does not read text brought dynamically in the DOM in firefox. I tested it with IE 11 it works ok, but with Firefox 
> 
> Example:
> 1. go to the following link and click on Update, it will not read the number that is being updated http://jsfiddle.net/fLzhR/89/
> 

### Expected result

> What did you expect to happen?

I expect JAWS to read the text when using firefox, with NVDA, it is reading fine 

### Actual result

> What actually did happen?

it did not read any text brought dynamically using JavaScript to the DOM 

### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

Please see it on this link http://jsfiddle.net/fLzhR/89/

#### JAWS version and build number

JAWS 18
 
#### Operating System and version

windows 7/10

#### Browser and version:
Firefox 57.0.1

**Comment by stevefaulkner** [comment]
@amad4biz Tested with Firefox 60.0b11 (64-bit) and JAWS 2018, could not replicate the issue. The updated number was announced by JAWS. Used this test case https://s.codepen.io/stevef/debug/xWeNXw

**Comment by amad4biz** [comment]
I just tested your code, but it is announcing the number in JAWS 18,  Firefox 57.0.1 (64 bit)

**Comment by ggordon-vfo** [comment]
Continues to work with JAWS 2019 and Firefox 67. Closing issue.
